packages/lib-classifier/src/components/Classifier/components/TaskArea/TaskArea.js

```diff
@@ -23,6 +25,7 @@ function storeMapper (stores) {
   }
 }
 
+@withTheme
```

Comment:
I had to double check the docs, but it says

> When using both `@inject` and `@observer`, make sure to apply them in the correct order: observer should be the inner decorator, inject the outer. There might be additional decorators in between.

So I think `@inject` needs to come before this?